cwms_radar_api/src/main/java/cwms/radar/ApiServlet.java

```diff
@@ -239,7 +255,7 @@ public static void radarCrud(@NotNull String path, @NotNull CrudHandler crudHand
             throw new IllegalArgumentException("CrudHandler requires a resource base at the beginning of the provided path, e.g. '/users/{user-id}'");
         }
 
-        Map<CrudFunction, Handler> crudFunctions = getHanders(crudHandler, resourceId);
+        Map<CrudFunction, Handler> crudFunctions = CrudHandlerKt.getCrudFunctions(crudHandler, resourceId);//getHanders(crudHandler, resourceId);
```

Comment:
I'm glad you got this working.  I could have sworn I tried to call the function this exact way.